boards/arm/stm32_min_dev/stm32_min_dev.yaml

```diff
@@ -6,3 +6,6 @@ toolchain:
   - zephyr
   - gnuarmemb
 ram: 20
+supported:
+  - gpio
```

Comment:
If you want to add gpio, do it in a separate commit.

Reply:
OK, I will send another PR